src/DocBlock/Annotation.php

```diff
@@ -232,7 +232,23 @@ public function setTypes(array $types)
     public function remove()
     {
         foreach ($this->lines as $line) {
-            $line->remove();
+            if ($line->isTheStart() && $line->isTheEnd()) {
```

Comment:
I can also make this line a `if (!( ... xor ...)) {` and then remove the else case (because the body is the same), however I chose to do it like this for readability and case separation.

If you want to go for the `! xor`, tell me and I'll change it.

Reply:
no xor please, operators like `or` or `and` have different prios than `||` or `&&`